templates/microprofile-kumuluzee/files/.platform.template.yaml

```diff
@@ -0,0 +1,53 @@
+  # An image URI (either base64-encoded or a URL) representing the template.
+
+  # Additional notes displayed in the template's detail view.
+  # Each note object is displayed as a small section heading with content below. Supports limited HTML.
+  notes:
+    - heading: "Apps & Services"
+      content: "Java 8<br/>Maven<br/>Eclipse MicroProfile<br/>KumuluzEE"
+
+
+# This key describes the initialization call made to the master environment at
+# project creation time. This is part of the full v2 UI operation mode, which
+# places project schema/options selection early in the creation process, rather
+# than later as it exitss now. To allow this schema to be backwards-compatible,
+# this key also gets mapped to the appropriate location in project.settings so
+# that the current UI can have its own workflow overridden as well.
+initialize:
+  repository: git://github.com/platformsh/template-microprofile-kumuluzee.git@master
+  config: null
+  files: []
+  profile: MicroProfile
```

Comment:
KumuluzEE?

Reply:
The profile is not the id.
If I can share the sam value I would like to use this value to all, once all are implementations of Eclipse MicroProfile:
https://microprofile.io/